app/src/main/res/layout/faq_activity.xml

```diff
@@ -1,7 +1,24 @@
 <?xml version="1.0" encoding="utf-8"?>
-<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<androidx.drawerlayout.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
+  xmlns:app="http://schemas.android.com/apk/res-auto"
   xmlns:tools="http://schemas.android.com/tools"
-  android:id="@+id/faq_fragment_placeholder"
+  android:id="@+id/faq_activity_drawer_layout"
   android:layout_width="match_parent"
-  android:layout_height="match_parent"
-  tools:context=".help.faq.FAQActivity" />
+  android:layout_height="match_parent">
+
+  <LinearLayout
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
+
+    <include
```

Comment:
The implementation in this should be similar to `exploration_activity`

Reply:
done